.github/workflows/yamllint.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: 'Checkout'
-        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+        uses: actions/checkout@v3
```

Comment:
can we pin yamllint action also?

Reply:
That one doesn't seem to promote this kind of pinning, can try tho.

Reply:
Nope, this one didn't work.